src/automations/checkInactivity.js

```diff
@@ -66,15 +73,9 @@ async function checkInactivePullRequest(pullRequest) {
   const repoOwner = pullRequest.base.repo.owner.login;
   const number = pullRequest.number;
 
-  const labels = await this.issues.getIssueLabels({
-    owner: repoOwner, repo: repoName, number: number
-  });
-  const inactiveLabel = labels.data.find(l => {
-    return l.name === this.cfg.activity.inactive;
-  });
-  const reviewedLabel = labels.data.find(l => {
-    return l.name === this.cfg.activity.pullRequests.reviewed.label;
-  });
+  const labels = await extractPRLabels(client, repoOwner, repoName, number);
```

Comment:
Why are you making an extra API call here? Thought it was already passed up in `scrapePullRequests()`

Reply:
This is because it needs to be called in `checkInactivePullRequest` irrespective of whether it is being called outside. It is called again in `scrapePullRequest` to extract the review status label which is being pushed into the `data` object.
That was the intent of putting the label extraction into a function so that it could be called in duplicate calls.